### PR TITLE
Simple workflow to run unit tests

### PR DESCRIPTION
**Pull request by bnmnetp**
This adds a workflow to run our unit tests and would allow us to remove Travis-CI.

Thanks to @timmartin for the test.yml file.

- [x] update README to add a badge.
- [ ] post accept update the protections on master to use this workflow instead of Travis.

The only real change to the repo is that we will add the package-lock.json file, which is recommended practice anyway, to the repo.

### CODE REVIEW

**Comment by bnmnetp** [comment]
It would be a nice addition if there was a way to create a workflow that updated our docs branch with a new version of skulpt.* after each merge to master.

**Comment by bnmnetp** [comment]
I don't think this requires much review.  Any @skulpt/committers have any objections?  If not I'll plan to merge this on Monday.

**Review by s-cork — APPROVED** [review]

**Pull request merged**
